### ISSUE TITLE
[Multirow taskbar for Windows 11] Serveral rows haven't taskbar alignment setting

### ISSUE THREAD

**Issue by bapohka** [ramensoftware/windhawk-mods#1338]
My taskbar alignment is 'center'
I set up `Multirow taskbar for Windows 11` for 2 rows.
I see that first row is aligned to right corner and second row aligned to left corner. Neither from it is aligned to center

Expect: both rows will have alignment to center and will be same size

![image](https://github.com/user-attachments/assets/b99d50c8-ea5c-452e-84e1-4094b022dd3d)

**Comment by m417z** [comment]
Yes, center alignment isn't supported properly, that's why the mod description says that the mod works best with left alignment. Perhaps I should have been more clear on that, using words such as "not supported" or "doesn't work" instead of "works best".

**Comment by m417z** [comment]
Yes, there's a similar request for the vertical taskbar as well: https://github.com/ramensoftware/windhawk-mods/issues/1068.
